frontend/src/pages/shipments/ShipmentCreatePage.tsx

```diff
@@ -18,7 +19,11 @@ const ShipmentCreatePage: FunctionComponent = () => {
 
   const onSubmit = (input: ShipmentCreateInput) => {
     // Create the shipment and then redirect to its "view" page
-    addShipment({ variables: { input } })
+    addShipment({
+      variables: { input },
+      // Fetch the updated list of shipments
+      refetchQueries: [{ query: AllShipmentsDocument }],
+    })
```

Comment:
Question (non-blocking): am I reading the docs right that if the `addShipment` mutation returned a list of all shipments with their IDs we wouldn't need a refetch query *or* update function?

Reply:
Yeah that seems to be the case but I feel a little uncomfortable with modifying the API in order to address a client-side implementation detail. I think I'll revisit this when I have a little more brain power.

Reply:
Glad to be learning about this stuff! I'm happy to adjust the API as we learn what will be most convenient for the UI.